src/helm/benchmark/run_expander.py

```diff
@@ -819,6 +819,92 @@ def expand(self, run_spec: RunSpec) -> List[RunSpec]:
             )
             for value in self.values
         ]
+    
+    
+class ChatMLRunExpander(RunExpander):
+    """
+    Adapt to ChatML: https://github.com/openai/openai-python/blob/main/chatml.md
+    A 1-shot example:
+    <|im_start|>system
+    Translate from English to French
+    <|im_end|>
+    <|im_start|>user
+    How are you?
+    <|im_end|>
+    <|im_start|>user
+    Comment allez-vous?
+    <|im_end|>
+    <|im_start|>user
+    {{user input here}}<|im_end|>
+    """
+    
+    name = "chatml"
+    
+    def __init__(self):
+        self.name = type(self).name
+
+    def expand(self, run_spec: RunSpec) -> List[RunSpec]:
+        adapter_spec = run_spec.adapter_spec
+        # according to https://github.com/openai/openai-python/blob/main/chatml.md#few-shot-prompting
+        # few-shot examples should do `<|im_start|>system name=example_user` or `<|im_start|>system name=example_assistant`
+        # but it is also possible to put examples into a user message.
+        
+        scenario_name = run_spec.name.split(':')[0]
+        
+        if scenario_name in ('msmarco',):
+            """
+            output_prefix:
+                Does the passage answer the query?
+                Answer:
+            
+            new_output_prefix:
+                Does the passage answer the query?<|im_end|>
+                <|im_start|>assistant
+                Answer:
+            """
```

Comment:
Could you use a comment block with `#` rather than a docstring, to be consistent with the existing style?